[2025-03-19T15:53:04.569+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_pipeline.update_external_data scheduled__2025-03-19T15:47:29.608366+00:00 [queued]>
[2025-03-19T15:53:04.592+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_pipeline.update_external_data scheduled__2025-03-19T15:47:29.608366+00:00 [queued]>
[2025-03-19T15:53:04.594+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-19T15:53:04.915+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): update_external_data> on 2025-03-19 15:47:29.608366+00:00
[2025-03-19T15:53:04.923+0000] {standard_task_runner.py:57} INFO - Started process 202 to run task
[2025-03-19T15:53:04.930+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_pipeline', 'update_external_data', 'scheduled__2025-03-19T15:47:29.608366+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_pipeline.py', '--cfg-path', '/tmp/tmp6c0piyi8']
[2025-03-19T15:53:04.935+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask update_external_data
[2025-03-19T15:53:05.007+0000] {task_command.py:415} INFO - Running <TaskInstance: fraud_detection_pipeline.update_external_data scheduled__2025-03-19T15:47:29.608366+00:00 [running]> on host f76c4cf6a8a2
[2025-03-19T15:53:05.148+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fraud_detection_pipeline' AIRFLOW_CTX_TASK_ID='update_external_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-19T15:47:29.608366+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-19T15:47:29.608366+00:00'
[2025-03-19T15:53:05.150+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-19T15:53:05.173+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=fraud_detection_pipeline, task_id=update_external_data, execution_date=20250319T154729, start_date=20250319T155304, end_date=20250319T155305
[2025-03-19T15:53:05.226+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-19T15:53:05.272+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
